### DISCUSSION TITLE
Changelog generation

### DISCUSSION THREAD

**Discussion by v1gnesh** [zopencommunity/meta#487] [Ideas]
Hey lookie here - https://github.com/apache/arrow-datafusion/blob/main/dev/release/generate-changelog.py

We're already using [PyGithub](https://github.com/PyGithub/PyGithub), so this will be a good fit.
Example: https://github.com/apache/arrow-datafusion/blob/main/dev/changelog/32.0.0.md
The way the Apache Arrow org does issues, PRs, epics is exemplary.

**Comment by MikeFultonDev** [comment]
Looks nice. We would need to start providing quality comments on commit, but that would be a good thing
